test/versions.ts

```diff
@@ -0,0 +1,50 @@
+export const versions = [
+	'Ubuntu 16.04.1',
+	'Ubuntu 14.04.4 LTS',
+	'Resin OS 2.0.9+rev1',
+	'Resin OS 2.0.8+rev1',
+	'Resin OS 2.0.7+rev1',
+	'Resin OS 2.0.6+rev4',
+	'Resin OS 2.0.6+rev3',
+	'Resin OS 2.0.6+rev2',
+	'Resin OS 2.0.6+rev1',
+	'Resin OS 2.0.5+rev1',
+	'Resin OS 2.0.5',
+	'Resin OS 2.0.4+rev2',
+	'Resin OS 2.0.4+rev1',
+	'Resin OS 2.0.3+rev1',
+	'Resin OS 2.0.2+rev2',
+	'Resin OS 2.0.0.rev1 (prod)',
+	'Resin OS 2.0.0-rc5.rev1',
+	'Resin OS 2.0.0-rc4.rev1',
+	'Resin OS 2.0.0-rc1.rev2',
+	'Resin OS 2.0.0-rc1.rev1',
+	'Resin OS 2.0.0-beta12.rev2',
+	'Resin OS 2.0.0-beta10.rev1',
+	'Resin OS 2.0.0-beta.8',
+	'Resin OS 2.0.0+rev4 (prod)',
+	'Resin OS 2.0.0+rev4 (dev)',
+	'Resin OS 2.0.0+rev3 (prod)',
+	'Resin OS 2.0.0+rev3 (dev)',
+	'Resin OS 2.0.0+rev3',
+	'Resin OS 2.0.0+rev2 (prod)',
+	'Resin OS 2.0-beta.8',
+	'Resin OS 1.8.0',
+	'Resin OS 1.6.0',
+	'Resin OS 1.26.0',
+	'Resin OS 1.24.1',
+	'Resin OS 1.24.0',
+	'Resin OS 1.22.0',
+	'Resin OS 1.21.0',
+	'Resin OS 1.2.1',
+	'Resin OS 1.19.0',
+	'Resin OS 1.16.0',
+	'Resin OS 1.1.4',
+	'Resin OS 1.1.1',
+	'Resin OS 1.1.0 (fido)',
+	'Resin OS 1.0.5 (fido)',
+	'Resin OS 1.0.0-pre',
+	'2.0.0.rev1',
+	null,
+	'foobar',
```

Comment:
Yeah, ok, fair enough. Should be a quick route to expanding these once we do that though, and it's nice clear documentation of how our ordering works in practice.